newserver_test.go

```diff
@@ -0,0 +1,46 @@
+package testing101
+import(
+	"bytes"
+	"testing"
+	"net/http"
+	"github.com/labstack/echo"
+	"net/http/httptest"
+)
+func TestPinging(t *testing.T) {
+	 e := echo.New()
+	 e.GET("/ping",Pinging)
+	req, err := http.NewRequest("GET", "/ping", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	rr := httptest.NewRecorder()
+	e.ServeHTTP(rr, req)
+	status := rr.Code
+	if status != http.StatusOK {
```

Comment:
Use[ tesify](https://godoc.org/github.com/stretchr/testify) library for assertion in tests. Use assertion makes your test automatic.